CONTRIBUTING.md

```diff
@@ -2,71 +2,20 @@
 
 We're excited you're considering contributing to our design system. If you have a question, noticed a bug, or have suggestions, then please submit an issue or create a pull request.
 
-One of our goals is to ensure a welcoming environment for all contributors. Please take a look at our [Code of Conduct](CODE_OF_CONDUCT.md) to learn more.
+One of our goals is to ensure a welcoming environment for all contributors. 
+Please take a look at our [Code of Conduct](CODE_OF_CONDUCT.md) to learn more.
+If you are interessted in running this site locally please take a look at [setting up your local development environment](/README.md#running-locally). 
 
-Below you'll find information on how to setup a local development environment and how to contribute code to the design system.
 
-### Additional information
-
-In **[our Wiki](https://github.com/CMSgov/design-system/wiki)** you can find additional information like:
+Use **[our Wiki](https://github.com/CMSgov/design-system/wiki)** to find additional information like:
 
 * Pattern proposal process
 * Coding guidelines
 * Guiding principles
 * How to write documentation
 * etc.
```

Comment:
I'm 💯 leaving this last bullet point if there'll be something added later. If not, it might be better to remove it.

Reply:
Let's remove in favor of where things are now 👍